cmd/osquery-perf/agent.go

```diff
@@ -242,7 +245,15 @@ type distributedReadResponse struct {
 	Queries map[string]string `json:"queries"`
 }
 
+func (a *agent) isOrbit() bool {
+	return a.deviceAuthToken != nil
+}
+
 func (a *agent) runLoop(i int, onlyAlreadyEnrolled bool) {
+	if err := a.orbitEnroll(); err != nil {
```

Comment:
Should we use `onlyAlreadyEnrolled` for Orbit?

This can be added later. 

Let's add docs or a TODO comment that if you kill the agent process then those Orbit node keys are gone, but maybe this is a-ok as Orbit will re-enroll?

Reply:
sounds good! I added a TODO comment and I will create an issue to make sure this doesn't get lost.